tests/users/test_models.py

```diff
@@ -117,6 +118,90 @@ def test_search_by_full_name(self):
         assert User.objects.search_by_full_name("Jean").count() == 2
         assert User.objects.search_by_full_name("Jean II").get() == user_1
 
+    def test_linked_job_seeker_ids(self):
+        organization = PrescriberOrganizationFactory()
+        other_organization = PrescriberOrganizationFactory()
+        prescriber = PrescriberMembershipFactory(organization=organization).user
+
+        # From the prescriber as no organization member
```

Comment:
Je trouve un poil plus claire cette formulation :
```suggestion
        # From the prescriber as member of no organization
```